_posts/wfh1972/wfh1972-2016-02-04-wfh1972.md

```diff
@@ -0,0 +1,28 @@
+--- 
+layout: post
+author: wfh1972
+title: "Wayne Homan's Chapter 5 Exercises"
+---
+
+<iframe src="https://trinket.io/embed/python/1ec9d4a40f" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+<iframe src="https://trinket.io/embed/python/d3f49a9d92" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+These programs were deceptively difficult for me. When I first looked at them I thought they would be simple; they weren't
+
+Exercise 1:
+I used x, y, and z as my sum, total number, and average variables.
+I used b as a dummy variable that would stop the while loop.
```

Comment:
Inventive.  Using `False` or `True` is more idiomatic, but checking the value of b with your comparison operator works just fine!